translations/harbour-advanced-camera-fi.ts

```diff
@@ -0,0 +1,300 @@
+        <source>Auto</source>
+        <translation>Auto</translation>
+    </message>
+    <message>
+        <source>Off</source>
+        <translation>Pois</translation>
+    </message>
+    <message>
+        <source>On</source>
+        <translation>Päällä</translation>
+    </message>
+    <message>
+        <source>Red Eye Reduction</source>
+        <translation>Punasilmäisyyden poisto</translation>
+    </message>
+    <message>
+        <source>Fill</source>
+        <translation>Täyttö</translation>
+    </message>
+    <message>
+        <source>Torch</source>
+        <translation>Soihtu</translation>
+    </message>
+    <message>
+        <source>Video Light</source>
+        <translation>Videovalo</translation>
+    </message>
+    <message>
+        <source>Slow Sync Front Curtain</source>
+        <translation>Etuverhon hidas synkronointi</translation>
+    </message>
+    <message>
+        <source>Slow Sync Rear Curtain</source>
+        <translation>Takaverhon hidas synkronointi</translation>
+    </message>
+    <message>
+        <source>Manual</source>
+        <translation>Manuaalinen</translation>
+    </message>
+</context>
+<context>
+    <name>FocusModel</name>
+    <message>
+        <source>Manual</source>
+        <translation>Manuaalinen</translation>
+    </message>
+    <message>
+        <source>Hyperfocal</source>
+        <translation>Hyperfokaalinen</translation>
+    </message>
+    <message>
+        <source>Infinity</source>
+        <translation>Ääretön</translation>
+    </message>
+    <message>
+        <source>Auto</source>
+        <translation>Auto</translation>
+    </message>
+    <message>
+        <source>Continuous</source>
+        <translation>Jatkuva</translation>
+    </message>
+    <message>
+        <source>Macro</source>
+        <translation>Makro</translation>
+    </message>
+    <message>
+        <source>Unknown</source>
+        <translation>Tuntematon</translation>
+    </message>
+</context>
+<context>
+    <name>GalleryUI</name>
+    <message>
+        <source>Deleting</source>
+        <translation>Poistetaan</translation>
+    </message>
+</context>
+<context>
+    <name>IsoModel</name>
+    <message>
+        <source>Auto ISO</source>
+        <translation>Auto ISO</translation>
+    </message>
+</context>
+<context>
+    <name>WbModel</name>
+    <message>
+        <source>Auto</source>
+        <translation>Auto</translation>
+    </message>
+    <message>
+        <source>Manual</source>
+        <translation>Manuaalinen</translation>
+    </message>
+    <message>
+        <source>Sunlight</source>
+        <translation>Auringonvalo</translation>
+    </message>
+    <message>
+        <source>Cloudy</source>
+        <translation>Pilvinen</translation>
+    </message>
+    <message>
+        <source>Shade</source>
+        <translation>Varjo</translation>
+    </message>
+    <message>
+        <source>Tungsten</source>
+        <translation>Keinovalo</translation>
```

Comment:
I think this should be "Hehkulamppu" or "Hehkuvalo" although some sites did use "Keinovalo"?